package.json

```diff
@@ -106,9 +106,9 @@
     "swagger-ui": "2.1.4",
     "tv4": "^1.2.7",
     "velocity-animate": "^1.2.3",
-    "vue": "1.0.24",
+    "vue": "1.0.26",
```

Comment:
Major breakage comes with 2.0 ;) I just read the changelog, there will be a little bit of migration work :/